KachnaOnline.App/Controllers/BoardGamesReservationsController.cs

```diff
@@ -0,0 +1,379 @@
+            }
+        }
+
+        /// <summary>
+        /// Updates an internal note in a reservation with the given ID.
+        /// </summary>
+        /// <param name="id">ID of the reservation to update.</param>
+        /// <param name="noteDto"><see cref="ReservationNoteInternalDto"/> containing the new internal note.</param>
+        /// <response code="204">The reservation was updated.</response>
+        /// <response code="404">No such reservation exists.</response>
+        [Authorize(Roles = RoleConstants.BoardGamesManager)]
+        [ProducesResponseType(204)]
+        [ProducesResponseType(404)]
+        [HttpPut("{id}/noteInternal")]
+        public async Task<ActionResult> UpdateReservationNoteInternal(int id, ReservationNoteInternalDto noteDto)
+        {
+            try
+            {
+                await _facade.UpdateReservationNoteInternal(id, noteDto);
+                return this.NoContent();
+            }
+            catch (ReservationNotFoundException)
+            {
+                return this.NotFound();
+            }
+            catch (ReservationManipulationFailedException)
+            {
+                return this.Problem(statusCode: 500);
+            }
+        }
+
+        /// <summary>
+        /// Creates a new reservation for a user.
+        /// </summary>
+        /// <param name="userId">ID of the user to create the reservation for.</param>
+        /// <param name="creationDto"><see cref="ManagerCreateReservationDto"/> to create.</param>
+        /// <returns>The created <see cref="ManagerReservationDto"/> if the creation succeeded.</returns>
+        /// <response code="201">The created reservation.</response>
+        /// <response code="404">When a requested game or user to create for does not exist.</response>
+        /// <response code="409">All of the given board games could not be reserved (e.g. are not available).
+        /// Returns the list of conflicting board game IDs.</response>
+        [Authorize(Roles = RoleConstants.BoardGamesManager)]
+        [ProducesResponseType(201)]
+        [ProducesResponseType(404)]
+        [ProducesResponseType(409)]
+        [HttpPost("madeFor/{userId}")]
+        public async Task<ActionResult<ManagerReservationDto>> ManagerCreateReservation(int userId,
+            ManagerCreateReservationDto creationDto)
+        {
+            try
+            {
+                var creatingUser = int.Parse(this.User.FindFirstValue(IdentityConstants.IdClaim));
+                var reservation = await _facade.ManagerCreateNewReservation(creatingUser, userId, creationDto);
+                return this.CreatedAtAction(nameof(this.GetReservation), new { id = reservation.Id }, reservation);
+            }
+            catch (BoardGameNotFoundException)
+            {
+                return this.NotFound();
+            }
+            catch (UserNotFoundException)
+            {
+                return this.NotFound();
```

Comment:
Hm, 422 could make sense too in this case. However since the user is part of the URI, in REST terms, the "resource", 404 sounds fine to me here as well